lib/team_collection.rb

```diff
@@ -17,4 +17,8 @@ def all_teams
     end
     team_objects
   end
+
+  def highest_scoring_visitor
+  x = @game_instances.highest_visitor_score
```

Comment:
Hey, can we change 'x' into a descriptive variable?

Reply:
I actually don't see this in my code?? At the top of the file it says outdated. What does that mean? I'm confused where that is coming from. I double checked and it's not in my branch file.

Reply:
Oh ^^ I'm seeing it's actually in our Master branch... I changed it in mine and I'll make the change with my PR